string gesture_name
int32 gesture_id
